book/website/reproducible-research/rdm/rdm-article.md

```diff
@@ -28,46 +28,56 @@ If you are planning to publish an analysis of your published data, it may be wor
 | Article type|  Cost estimate |
```

Comment:
The text above this states that the cost information was collected in Feb 2022, is that still the case?

Reply:
Yes, this is not updated

Reply:
July 2023! Great spot :)